contracts/http/response.go

```diff
@@ -37,6 +39,10 @@ type ContextResponse interface {
 	View() ResponseView
 	// Writer returns the underlying http.ResponseWriter associated with the response.
 	Writer() http.ResponseWriter
+	// WithCookie adds a cookie to the response.
+	WithCookie(cookie *Cookie) ContextResponse
+	// WithoutCookie removes a cookie from the response by setting its expiration time to a past date.
```

Comment:
Drivers can remove cookie by themselves, maybe not `setting its expiration time to a past date`

```suggestion
	// WithoutCookie removes a cookie from the response.
```

Reply:
Yeah, you are right.